src/drivers/distance_sensor/broadcom/afbrs50/AFBRS50.hpp

```diff
@@ -89,4 +89,7 @@ class AFBRS50 : public px4::ScheduledWorkItem
 	hrt_abstime _measurement_time{0};
 
 	perf_counter_t _sample_perf{perf_alloc(PC_INTERVAL, MODULE_NAME": sample interval")};
+
+	float max_distance;
+	float min_distance;
```

Comment:
Hi @samchamberlin , these changes look good.  Just one note, to follow convention, class member variables in this project typically lead with an underscore, e.g. `_max_distance` and local variables do not lead with an underscore.  If you can fix things up to meet the convention I think everything is ship shape!  Good work!

Reply:
Thanks @mcsauder, updated the variable names!